packages/spec-test-runner/test/spec/altair/fork_choice/get_head_minimal.test.ts

```diff
@@ -0,0 +1,207 @@
+        if (!signedBlock) throw Error(`No block ${step.block}`);
+        expect(signedBlock).not.to.be.undefined;
+        const preState = stateCache.get(toHexString(signedBlock.message.parentRoot));
+        if (!preState)
+          throw new Error("not found parent state for parent root" + toHexString(signedBlock.message.parentRoot));
+        wrappedState = runStateTranstion(preState, signedBlock, forkchoice, checkpointStateCache);
+        cacheState(wrappedState, stateCache);
+      } else if (isCheck(step)) {
+        const {
+          head: expectedHead,
+          time: expectedTime,
+          justifiedCheckpointRoot,
+          finalizedCheckpointRoot,
+          bestJustifiedCheckpoint,
+        } = step.checks;
+        const head = forkchoice.getHead();
+        expect(head.slot).to.be.equal(Number(expectedHead.slot));
+        expect(toHexString(head.blockRoot)).to.be.equal(expectedHead.root);
+        // time in spec mapped to Slot in our forkchoice implementation
+        if (expectedTime) expect(forkchoice.getTime() * SECONDS_PER_SLOT).to.be.equal(Number(expectedTime));
+        if (justifiedCheckpointRoot)
+          expect(toHexString(forkchoice.getJustifiedCheckpoint().root)).to.be.equal(justifiedCheckpointRoot);
+        if (finalizedCheckpointRoot)
+          expect(toHexString(forkchoice.getFinalizedCheckpoint().root)).to.be.equal(finalizedCheckpointRoot);
+        if (bestJustifiedCheckpoint)
+          expect(toHexString(forkchoice.getBestJustifiedCheckpoint().root)).to.be.equal(bestJustifiedCheckpoint);
+      }
+    }
+  },
+  {
+    inputTypes: {
+      meta: InputType.YAML,
+      steps: InputType.YAML,
+    },
+    sszTypes: {
+      [ANCHOR_STATE_FILE_NAME]: config.types.altair.BeaconState,
+      [ANCHOR_BLOCK_FILE_NAME]: config.types.altair.BeaconBlock,
+      [BLOCK_FILE_NAME]: config.types.altair.SignedBeaconBlock,
+      [ATTESTATION_FILE_NAME]: config.types.phase0.Attestation,
+    },
+    mapToTestCase: (t: Record<string, any>) => {
+      // t has input file name as key
+      const blocks = new Map<string, altair.SignedBeaconBlock>();
+      const attestations = new Map<string, phase0.Attestation>();
+      for (const key in t) {
+        const blockMatch = key.match(BLOCK_FILE_NAME);
+        if (blockMatch) {
+          blocks.set(key, t[key]);
+        }
+        const attMatch = key.match(ATTESTATION_FILE_NAME);
+        if (attMatch) {
+          attestations.set(key, t[key]);
+        }
+      }
+      return {
+        meta: t["meta"] as IForkChoiceTestCase["meta"],
+        anchorState: t[ANCHOR_STATE_FILE_NAME] as IForkChoiceTestCase["anchorState"],
+        anchorBlock: t[ANCHOR_BLOCK_FILE_NAME] as IForkChoiceTestCase["anchorBlock"],
+        steps: t["steps"] as IForkChoiceTestCase["steps"],
+        blocks,
+        attestations,
+      };
+    },
+    timeout: 10000000,
+    // TODO: fix this
+    shouldSkip: (testCase, name) => name === "filtered_block_tree",
```

Comment:
Could you detail the issue with this test and the exact error that breaks it?

Reply:
just open https://github.com/ChainSafe/lodestar/issues/2567